source/documentation/runbooks/012-support-duties.html.md.erb

```diff
@@ -14,6 +14,17 @@ Support is provided on weekdays between 9am - 5pm.
 
 ## Duties
 
+- Monitor the **FindMoJData@justice.gov.uk** shared mailbox for any issues that have been reported with data
+>**Note**: You will first need to request permissions via servicenow before adding the mailbox within the Microsoft outlook client
```

Comment:
Can you include a link here to the service now form?

Reply:
That is my plan but I am unable to access snow myself currently.